agents/ppo/ppo.py

```diff
@@ -0,0 +1,52 @@
+from agents.ppo.types import *
+from typing import List
+import torch
+
+
+class PPOAgent(TorchAgent, config_name='ppo'):
+    def __init__(self, actor: BaseActor, critic: BaseCritic, discount_factor: float, ratio_clip: float):
```

Comment:
There are more than three arguments in this line. Let's re-write it more or less like this:

```suggestion
    def __init__(
        self, actor: BaseActor, 
        critic: BaseCritic, 
        discount_factor: float, 
        ratio_clip: float
    ):
```